pkg/mediationcontainer/client_websocket_transport.go

```diff
@@ -256,6 +299,23 @@ func (clientTransport *ClientWebSocketTransport) performWebSocketConnection() er
 	return errors.New("Abort client socket connect, transport is closed")
 }
 
+// set up websocket Ping-Pong protocol handlers
+func setupPingPong(ws *websocket.Conn) {
+	h := func(message string) error {
+		glog.V(3).Infof("Recevied ping msg")
+		ws.WriteControl(websocket.PongMessage, []byte(message), time.Now().Add(writeWaitTimeout))
```

Comment:
This can throw error. Why not using the default one, where the error is handled with some logic? Is it because we want to log the message?

Reply:
Yes. Main purpose is to add the log. So that we can know the status of the websocket.

The default ping-handler will return error. This one won't report error.  I can replace it with the default one, and the log.

Reply:
Currently, the Turbo server won't send any ping msg.  
We only need the pong-handler, to make sure that server has received our ping msg.